packages/desktop-gui/cypress/integration/settings_spec.js

```diff
@@ -581,48 +588,6 @@ describe('Settings', () => {
     })
   })
 
-  describe('errors', () => {
```

Comment:
Why delete these tests?

Reply:
They looked to be duplicates. The same error test is at the bottom of the file.